shepherd_bms/include/segment.h

```diff
@@ -1,7 +1,115 @@
 #ifndef SEGMENT_H
 #define SEGMENT_H
 
+#include <LTC68041.h>
+#include <nerduino.h>
 #include "datastructs.h"
 
+#define NUM_SEGMENTS    4
+#define NUM_CHIPS       NUM_SEGMENTS*2
+
+#define THERM_REF       4700            // Reference resistor for thermistor measurments / voltage divider
```

Comment:
Do we need this?

Reply:
Ngl i wasn't entirely sure what it was being used for, but it was added by you in the testing code so I put it in just in case

Reply:
```suggestion
```

Reply:
I would keep it honestly. Idk bout others, but for me having the code reflect the pcb design helps bring together the shepherd project in my head